mava/utils/make_env.py

```diff
@@ -199,6 +215,41 @@ def make_gigastep_env(
     return train_env, eval_env
 
 
+def make_gym_env(
+    config: DictConfig,
+    num_env: int,
+    add_global_state: bool = False,
+) -> gymnasium.vector.AsyncVectorEnv:
+    """
+     Create a gymnasium environment.
+
+    Args:
+        config (Dict): The configuration of the environment.
+        num_env (int) : The number of parallel envs to create.
+        add_global_state (bool): Whether to add the global state to the observation. Default False.
+
+    Returns:
+        Async environments.
+    """
+    base_env_name = config.env.scenario.name
+    env_maker, wrapper = _gym_registry[base_env_name]
```

Comment:
`base_env_name ` is used once you can skip creating a variable for it:

```suggestion
    env_maker, wrapper = _gym_registry[config.env.scenario.name]
```
`